EKF/ekf_helper.cpp

```diff
@@ -188,6 +188,12 @@ void Ekf::resetHeight()
 
 	}
 
+	// zero the cross covarinace entries for height and vertical velocity
```

Comment:
Row and column 8 should only be zeroed if vert_pos_reset is true

Similarly for Row and column 5 and vert_vel_reset.